lib/circleci/project.rb

```diff
@@ -130,6 +130,19 @@ def self.settings username, project
       CircleCi.http.get "/project/#{username}/#{project}/settings"
     end
 
+    ##
+    #
+    # Delete a checkout key
+    #
+    # @param username     [String] - User or org name who owns project
+    # @param project      [String] - Name of project
+    # @param fingerprint  [String] - Fingerprint
+    # @return         [CircleCi::Response] - Response object
```

Comment:
Correct the spacing here please to match the other indentation of parameter types.